test/server/config/http/config_test.cc

```diff
@@ -398,13 +398,13 @@ TEST(HttpFilterConfigTest, RouterFilterWithEmptyProtoConfig) {
   cb(filter_callback);
 }
 
-TEST(HttpFilterConfigTest, IpTaggingFilter) {
+TEST(HttpFilterConfigTest, IpTaggingFilterV1Config) {
   std::string json_string = R"EOF(
   {
-    "request_type" : "internal",
+    "request_type" : "INTERNAL",
```

Comment:
Why the switch?   I'm a bit twitchy about changing any config, even if we expect no one is using it in production.

Reply:
To be consistent between the V2 enums and V1 enums.

Reply:
My vote is to just delete v1 support for this new feature. I don't think it's worth adding. But I don't feel that strongly about it.